config/production.yaml

```diff
@@ -3,7 +3,7 @@
   CONTRIBUTOR_EMAIL:
     transport:
       debug: false
-      logger: false
```

Comment:
Is that going to log a contributor's personal email address to Splunk?

Reply:
(I think it would make GDPR sad)

Reply:
I doubt it, it looks to be the name of the variable https://github.com/Financial-Times/next-syndication-api/blob/master/worker/sync/db-persist/mail-contributor.js#L14